src/redux/reducers/experimentSettings/initialState.js

```diff
@@ -1,6 +1,6 @@
 export default {
   processing: {
-    meta: {
+    processingConfig: {
```

Comment:
Can you please return this key back to `meta`? It's purposefully called `meta` to describe metadata about file data processing (e.g. progress, status (loaded, error), etc.)